from exceptions import convert

if __name__ == "__main__":
    convert("two nine".split())